generated/ruby/test/errorreporting.rb

```diff
@@ -0,0 +1,160 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Integration tests for vkit generated client library for
+# Stackdriver ErrorReporting APIs.
+#
+require "google/cloud/error_reporting/v1beta1"
+
+PROJECT_NAME = "gapic-test".freeze
+SERVICE = "default".freeze
+VERSION = "20160920t135819".freeze
+USER_NAME = "testuser".freeze
+
+#
+# Integration test for ReportErrorsServiceApi.report_error_event
+#
+# Report first two error events (to be one group) and third event (to be another group)
+ReportErrorsServiceApi =
+  Google::Cloud::ErrorReporting::V1beta1::ReportErrorsServiceApi
+report_errors_service_api = ReportErrorsServiceApi.new
+Grpc = Google::Devtools::Clouderrorreporting::V1beta1
+ReportedErrorEvent = Grpc::ReportedErrorEvent
+ServiceContext = Grpc::ServiceContext
+ErrorContext = Grpc::ErrorContext
+HttpRequestContext = Grpc::HttpRequestContext
+SourceLocation = Grpc::SourceLocation
+
+
+formatted_project_name = ReportErrorsServiceApi.project_path(PROJECT_NAME)
+http_request_context = HttpRequestContext.new(
+  method: "GET",
+  url: "https://20160920t135819-dot-gapic-test.appspot.com")
```

Comment:
Does this URL have to be an appspot.com URL? Since it's a test, some "example.com" URL might make sense if that's okay.